.jonathan-troop

```diff
@@ -0,0 +1,225 @@
+
+    30. How do I write a comment in CSS?
+
+    start a comment with /* and end it with */
+
+    31. How do I find out what file I am in, when I am using the command line? 
+
+    pwd
+
+    32. Using the command line - how do I see a list of files/folders in my current folder?
+
+    ls
+
+    33. How do I remove a file via the command line? Why do I have to be careful with this? 
+
+    rm <file>
+
+    34. Why should I use version control? 
+
+    It helps to organize the coding process, especially among many members in a team. 
+    This way, each member can track the changes they made, in what order, and which changes got integrated into the main branch.
+
+
+    35. How often should I commit to github?
+
+    As often as possible -- with completed tasks
+
+    36. What is the command we would use to push our repo up to github? 
+
+    1: git add .
+    2: git commit -m 'comments'
+    3. git push 
+
+    37. Walk me through Lambda's git flow. 
+
+    First you fork the repo from github. Then you add your TL as a collaborator. Then you navigate to a directory in terminal and execute git clone "url".
+    Then you make a branch for yourself, git checkout -b 'firstName-lastName'. Then you make whatever changes necessary as per the project guidelines and push it to 
+    the repo using the steps in #36. After that, go to pull requests on github and select your TL as a reviewer for your git. 
+
+Stretch Questions
+
+    1. What is the difference between an inline element and a block element?
+
+    A block element starts on a line and takes up the entire space. examples include <p> <footer> and <div>.
+
+    An inline element doesn't start a new line and only takes up as much width as it uses. exmaples include <span> <button> and <img>
+
+    2. What happens when an element is positioned absolutely? 
+
+    The element is positioned to its nearest closest relative, but does not have to adhere to the positioning of everytthing else in that section. It also gives 
+    the flexibility to position the element wherever you want to display it, even overlapping other elements. 
+
+    3. How do I make an element take up only the amount of space it needs but also have the ability to give it a width? 
+
+    I would identify the element the content is in, and add a width property in css to it. I could set it to a fixed width, such as 600px, or a % of the width (or a mix of both). 
+    This should ensure that the conent will fit within a width and wrap down to fill itself as necessary.
+
+    4. Name 3 elements that are diplay block by default, 2 elements that are display inline by default and 1 element that is display inline-block by default
+
+    Display block: <table></table> <nav></nav> and <form></form> 
+    Display inline: <script></script> <strong></strong> and <a></a> 
+    Display inline-block: none, from what I could come up with. It is a CSS property and allows to give width and height to an inline element. Often used for
```

Comment:
button is an inline-block element, as it only takes up as much space as it needs yet you can still set its width